app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -59,6 +59,7 @@ import org.oppia.app.application.ApplicationModule
 import org.oppia.app.application.ApplicationStartupListenerModule
 import org.oppia.app.help.HelpActivity
 import org.oppia.app.options.OptionsActivity
+import org.oppia.app.player.state.StateFragmentTest
```

Comment:
Do we need this import? If we are reusing something from StateFragmentTest, lets move that part to the testing module and have StateFragmentTest and this test use that

Reply:
Good catch, Thanks

Reply:
LGTM now, resolving this